data_steward/analytics/cdr_ops/rt_cdr_qc/cdr_deid_qa_report10_extra.py

```diff
@@ -61,17 +61,27 @@
 query = JINJA_ENV.from_string("""
 
 SELECT COUNT(*) as n_participants_over_89 FROM `{{project_id}}.{{rt_cdr_deid}}.person`
-WHERE FLOOR(DATE_DIFF(DATE('{{cdr_cutoff_date}}'),DATE(birth_datetime), DAY)/365.25) > 89
+WHERE FLOOR(DATE_DIFF(DATE('{{cdr_cutoff_date}}'),DATE(birth_datetime), DAY)/365.25) > {{max_age}}
 """)
-q = query.render(project_id=project_id,rt_cdr_deid=rt_cdr_deid,cdr_cutoff_date=cdr_cutoff_date)
-
-df1=execute(client, q)
-if df1.loc[0].sum()==0:
- df = df.append({'query' : 'Query1 No person exists over 89 in the dataset', 'result' : 'PASS'},  
-                ignore_index = True) 
+q = query.render(project_id=project_id,
+                 rt_cdr_deid=rt_cdr_deid,
+                 cdr_cutoff_date=cdr_cutoff_date)
```

Comment:
max age needs to be added here.
 `cdr_cutoff_date=cdr_cutoff_date, 
                 max_age=max_age)`